cmd/thv-operator/pkg/registryapi/config/config.go

```diff
@@ -244,7 +261,12 @@ func (c *Config) ToConfigMapWithContentChecksum(mcpRegistry *mcpv1alpha1.MCPRegi
 const DefaultRegistryName = "default"
 
 func (cm *configManager) BuildConfig() (*Config, error) {
-	config := Config{}
+	config := Config{
+		// default to anonymous authentication until we model it consistently in the MCPRegistry CRD
+		Auth: &AuthConfig{
+			Mode: AuthModeAnonymous,
```

Comment:
I think we should default to `oauth` and allow users to disable by specifying `anonymous`